.github/workflows/build-docker-artifact.yaml

```
@@ -46,7 +59,7 @@ on:
             - "amd64"
 
 env:
-  IMAGE_NAME: ${{ inputs.distro }}-${{ inputs.version }}-${{ inputs.architecture }}
+  IMAGE_NAME: ${{ inputs.distro }}-${{ inputs.version }}-${{ inputs.target }}-${{ inputs.architecture }}
```

Simplest solution and easiest to adjust when @tt-rkim revamps this workflow:

```suggestion
  IMAGE_NAME: ${{ inputs.distro }}-${{ inputs.version }}-dev-${{ inputs.architecture }}
```